src/circuit/circuit_frame.rs

```diff
@@ -1805,6 +1797,57 @@ fn reduce_cons<F: LurkField, CS: ConstraintSystem<F>>(
     /////////////////////////////////////////////////////////////////////////////
     results.add_clauses_cons(*if_hash.value(), &arg1, env, &newer_cont, &g.false_num);
 
+    // head == CALL and CALL0, newer_cont is allocated
```

Comment:
This comment doesn't make sense to me. Other similar comments are specifying the value of the symbol in the head position of the source code, but neither CALL nor CALL0 name source symbols.

Reply:
Let's try to figure out a way to communicate that information that's consistent with the rest of the comments.